### ISSUE TITLE
Translate comments from Swedish to English

### ISSUE THREAD

**Issue by esaruoho** [kurasu/surge#67]
As @kurasu warned, a couple of comments were in Swedish. So I translated them to English, with Google Translate, some work by hand, and a lot of help by a friend.

Check the PR at #68

**Comment by baconpaul** [comment]
This translation would be super useful!

**Comment by esaruoho** [comment]
@baconpaul i'm working on it.. i've got a branch in my fork

**Comment by baconpaul** [comment]
Thank you! A good week here - we went from a million compiler errors to a sorta working ui. Been fun hacking through it with you!

**Comment by esaruoho** [comment]
geez i don't know how to find all the  ISO-8859-1 encoded files. i'm gonna make this into a PR and see..

**Comment by esaruoho** [comment]
@baconpaul here's a start of translations, #68 

I need to figure out some magick on OSX to find which files are encoded in ISO-8859-1 so I can find @kurasu 's Swedish language comments. I'll do that tomorrow. Hope this helps anyway.

**Comment by esaruoho** [comment]
if someone can help me figure out how to use

`file -l` on OSX recursively on the whole repo, that would really help.

**Comment by baconpaul** [comment]
```
find src -exec file -I {} \; | grep -v inode/directory | grep -v us-ascii
 ```

and that's -I (capital i) not -l (lower case L)

Attached output as a txt file

[f.txt](https://github.com/kurasu/surge/files/2677976/f.txt)

**Comment by esaruoho** [comment]
thanks @baconpaul - that helped immensely. i'll check through the remainder of the files in the next few hours.

**Comment by esaruoho** [comment]
Thanks @baconpaul - I've now finished with the PR translations, so will tag it as one that closes this ticket.

**Comment by esaruoho** [comment]
Closed by #92
